packages/manager/apps/pci-savings-plan/src/components/CreatePlanForm/CreatePlanForm.tsx

```diff
@@ -317,7 +326,11 @@ const CreatePlanForm: FC<CreatePlanFormProps> = ({
           <OdsQuantity
             onOdsChange={(event: OdsInputChangeEvent) => {
               const newValue = Number(event.detail.value);
-              if (newValue >= 1 && newValue <= 1000) onChangeQuantity(newValue);
+              if (newValue >= 1 && newValue <= 1000) {
+                onChangeQuantity(newValue);
+              } else {
+                onChangeQuantity(1);
+              }
```

Comment:
```suggestion
              onChangeQuantity(newValue >= 1 && newValue <= 1000 ? newValue : 1);
```
This is a quick refactor but I would suggest to create a function to normalize the value outside the component to make it more understandable since I don't understand why 1 and 1000 are the limits

Reply:
BTW, please avoid putting logic directly in JSX, prefer using a dedicated function as @SimonChaumet said 👍🏻